### ISSUE TITLE
Requests to manifest.json do not respect cookies and headers

### ISSUE THREAD

**Issue by freender** [Radarr/Radarr#5863]
Requests to manifest.json do not respect cookies and headers

Prerequisites:
NGINX reverse proxy + oauth2_proxy (for 2FA)
TC:
1. Login to Radarr
2. User receives _oauth2_proxy cookie from Google API
3. Reverse proxy redirects user to Radarr
4. On main Radarr page Right click -> Inspect -> Network -> Refresh

**Actual Result:**

Radarr sends Get request for manifest.json without cookies. As a result request blocked by reverse proxy
/Content/Images/Icons/manifest.json?h= ...

```
frontend/src/index.html
<link rel="manifest" href="/Content/Images/Icons/manifest.json" />

frontend/src/login.html
<link rel="manifest" href="/Content/Images/Icons/manifest.json" />

```

**Expected Result** As discussed on links below this can be fixed by adding crossorigin="use-credentials" in the manifest tag.

```
frontend/src/index.html
<link rel="manifest" href="/Content/Images/Icons/manifest.json" crossorigin="use-credentials" />


frontend/src/login.html
<link rel="manifest" href="/Content/Images/Icons/manifest.json" crossorigin="use-credentials" />
```

https://github.com/Sonarr/Sonarr/issues/4305
https://github.com/morpheus65535/bazarr/issues/1084
https://stackoverflow.com/questions/51777346/cookies-not-sent-with-request-for-web-app-manifest-json
https://web.dev/add-manifest/

**Comment by bakerboy448** [comment]
Sonarr Xref https://github.com/Sonarr/Sonarr/issues/4305  @freender can you xref the sonarr issue if you open on readarr and lidarr?

we'll likely have markus fix it and let it flow downstream :D

**Comment by freender** [comment]
Yep I will :)

**Comment by Qstick** [comment]
We actually fixed this in V0.2... It just never made it to 3 <https://github.com/Radarr/Radarr/commit/69811b1291407235195316ea33764596e866bc8a>
